apps/common/src/main/scala/org/lfdecentralizedtrust/splice/config/SpliceConfig.scala

```diff
@@ -21,6 +21,8 @@ abstract class SpliceBackendConfig extends LocalNodeConfig {
   override val topology: TopologyConfig = TopologyConfig()
 
   override val monitoring: NodeMonitoringConfig = NodeMonitoringConfig()
+
+  val latestPackagesOnly: Boolean = false
```

Comment:
afaict the default parameter here does not trigger during config parsing and the one that actually matters is the one in SvAppConfig and similar. That seems to be the reason why CI is completely broken due to config parsing failures.